requestor/requirements.txt

```diff
@@ -1,7 +1,5 @@
---extra-index-url https://test.pypi.org/simple/
-
 Quart==0.15.1
 quart-cors==0.5.0
+web3==5.20.0
 
-#   TODO - wait for yapapi release
-yapapi~=0.6.0a
+git+https://github.com/golemfactory/yapapi-service-manager.git@johny-b/final-touch-APPS-177
```

Comment:
This needs to be updated to `master` once you merge https://github.com/golemfactory/yapapi-service-manager/pull/5#pullrequestreview-688351990